### PR TITLE
Feat(home): Added index.js and isPrime.js

### PR DESCRIPTION
**Pull request by syukronarie**

### CODE REVIEW

File: isPrime.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+if (number === 1) {
+	console.log("1 is neither prime nor composite number.");
+}
+
+// check if number is greater than 1
+else if (number > 1) {
+	// looping through 2 to number-1
+	for (let i = 2; i < number; i++) {
+		if (number % i == 0) {
+			isPrime = true;
```
Comment (muhammhafizh) [inline isPrime.js]: Test

Comment (Bimahayunugraha) [inline isPrime.js]: Assigning value should be true not false

Comment (afauzann) [inline isPrime.js]: Assigning value should be true not false

Comment (syukronarie) [inline isPrime.js]: Assigning value should be true not false

Comment (riorasyid23) [reply isPrime.js]: test

File: isPrime.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// program to check if a number is prime or not
+
+// take input from the user
+const number = parseInt(prompt("Enter a positive number: "));
+let isPrime = true;
```
Comment (hariefairuzzaki) [inline isPrime.js:5]: assigning value should true not false

File: isPrime.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+// check if number is equal to 1
+if (number === 1) {
+	console.log("1 is neither prime nor composite number.");
+}
```
Comment (anangkf) [inline isPrime.js:10]: value should be true not false

File: isPrime.js (hunk 1 above)
Comment (muhammhafizh) [reply isPrime.js]: test

Comment (mochammadilham06) [reply isPrime.js]: test comment

**Review by anangkf — CHANGES_REQUESTED** [review]

File: isPrime.js (hunk 1 above)
Comment (EagleEyes1) [reply isPrime.js]: test

Comment (juanprdsi) [reply isPrime.js]: testing

**Review by takimsry — APPROVED** [review]

**Review by anangkf — APPROVED** [review]

**Review by afauzann — APPROVED** [review]

**Review by muhammhafizh — COMMENTED** [review]
test

**Review by afauzann — COMMENTED** [review]
ok

**Pull request merged**